ModelCatalogueCorePlugin/src/groovy/org/modelcatalogue/core/util/Metadata.groovy

```diff
@@ -14,5 +14,6 @@ class Metadata {
      * The HPO and clinical tests report exporters only work for and should only be available to the relevant data class(es), discerned by this metadata field
      */
     public static final def HPO_REPORT_AVAILABLE = "rare-disease-hpo-clinical-tests-report"
+    public static final def CANCER_TYPES_AVAILABLE = "cancer-types-export"
```

Comment:
from my point of view
1. URL gives the user better understanding that this metadata value is something special and it is going to be processed by the catalogue
2. minimize the name clash and applying this flag by accident
3. if the URL points to the webpage the webpage can contain more information how does the flag work
4. this is a way how metadata with special meaning are declared across the catalogue (see e.g. CRF exporter or new metadata on data model)


I can live with what you've submitted but to be coherent across the catalogue I would prefer to change the key to some url like `http://www.genomicsengland.co.uk/metadata/cancer-types-export` or similar.